string id
string cmd
bool success
string msg
uint8[] data